### ISSUE TITLE
Parameterizing Single and Double Pendulum Examples

### ISSUE THREAD

**Issue by cd155** [JacquesCarette/Drasil#2801]
In the nth pendulums, we parameterize masses and rods. In addition, more derivatives will be parameterized (ith part), such as velocity, acceleration, angle, speed of angle, acceleration of angle. However, when we work on the single pendulum, we don't want to parameterize each variable. This is the ideal situation, but it hasn't be done in this way.

In the double pendulum example, there will be 1 and 2 subscripts in most elements. They are indicating variables relate to the first mass and the second mass. In the single pendulum example, there will be no subscript because it is a natural way to not have subscript when each element only has quantity in one. This raises the problem that some duplicates occur between two examples. 

In the double pendulum example, some definitions relate to the first mass are almost identical with its counterpart in the single pendulum example. For example, the equation of calculating the velocity in the horizontal direction in the first mass is almost identical to the equation of calculating the velocity in the horizontal direction in its only mass. The only difference is the subscript. They are more in these two examples. If we think about solving the nth pendulum, we could subscript between 1th to nth. Technically, it is not wrong to put 1 subscript in the single pendulum, but it is just not a natural way we present it.

Ideally, the ultimate goal is that removing those duplicate knowledge.

Here are all duplicates in the Single and Double Pendulum Example.

pending

relate to #2730

**Comment by JacquesCarette** [comment]
This seems to intersect with what @tingyuw was talking about this morning, and where @smiths was suggesting some name changes to get around double-superscripts (where here it's double-subscripts). We really shouldn't use double-sub/superscripts!

**Comment by smiths** [comment]
@cd155 for velocity you can use u for v_x and v for v_y.  For the position you sometimes see a and b for p_x and p_y, respectively.  The acceleration is a tough one.  For acceleration I'm not sure how to replace a_x and a_y.  Maybe do a bit of googling and see what others use to represent the components of the acceleration vector?

**Comment by cd155** [comment]
@smiths @JacquesCarette Thanks for pointing it out. I can search around to see how other people represent them. Just want to double-check, in the situation listed above, should we just use another alphabet to avoid double sup/subscript.

**Comment by smiths** [comment]
Yes, we will aim to use a notation that avoids the double sup/subscripts.  However, we need to pick a notation that is still commonly used.  We don't want to introduce something that is arbitrary.  It might be technically correct, but it will be hard for domain experts to understand.

In some cases a double subscript or superscript could be fine, like if we were referencing the ith row and jth column of matrix A (A_{ij}).  It isn't always a bad thing.  :smile:
